src/main/java/frc/robot/subsystems/ClimberSubsystem.java

```diff
@@ -8,43 +8,29 @@
 
 public class ClimberSubsystem extends SubsystemBase {
 
-  // This is never deactivated, only activated. Climber is one and done.
+  // This is never deactivated after use. It exists to turn on.
   private final Solenoid climberSV =
       new Solenoid(Constants.SOLENOID_MODULE_TYPE, Constants.CLIMBER_SOLENOID_CHANNEL);
 
-  /** Constructor for this subsystem. used to create an object in RobotContainer. */
+  /** ClimberSubsystem constructor */
```

Comment:
🎨 I personally feel that these sorts of comments don't provide value. It is immediately apparent that this is a constructor. I would just remove it.